src/v2/controllers/FormController.js

```diff
@@ -133,6 +133,11 @@ export default Controller.extend({
   handleInvokeAction(actionPath = '') {
     const idx = this.options.appState.get('idx');
 
+    // TODO: use useInteractionCodeFlow from settings instead after OKTA-405474
+    const authClient = this.options.settings.getAuthClient();
+    const transactionMeta = authClient.transactionManager.load();
```

Comment:
Because `useInteractionCodeFlow` is unset after starting login flow here:
https://github.com/okta/okta-signin-widget/blob/56930aa682bf8e0aee294b5a1c8588c5b0d4c6a8/src/v2/BaseLoginRouter.js#L218
The only way I see to "compute" this property after being deleted is to check `interactionHandle` in transaction meta.
And to load transaction meta we need to use auth-js's manager: 
https://github.com/okta/okta-signin-widget/blob/56930aa682bf8e0aee294b5a1c8588c5b0d4c6a8/src/v2/client/interactionCodeFlow.js#L22